topics/ecology/tutorials/lotus2-identifying-fungi/tutorial.md

```diff
@@ -0,0 +1,462 @@
+---
+layout: tutorial_hands_on
+
+title: Identifying Mycorrhizal Fungi from ITS2 sequencing using LotuS2
+
+zenodo_link: https://zenodo.org/records/13710591
+
+abbreviations:
+  SPUN: Society for the Protection of Underground Networks
+  MF: Mycorrhizal fungi
+  EcMF: ectomycorrhizal fungi
+  AMF: arbuscular mycorrhizal fungi
+  eDNA: environmental DNA
+  ITS2: Internal Transcribed Spacer 2
+  SSU: Small Subunit (RNA)
+  OTU: Operating Taxonomic Unit
+  ASV: Amplicon Sequence Variant
+  sdm: simple demultiplexer
+  TSV: tab-separated-values
+
+requirements:  
+- 
+  type: "internal"  
+  topic_name: introduction  
+  tutorials:  
+    - galaxy-intro-101-everyone  
+
+questions:
+- What is the fungal community composition in a given soil sample?
+
+objectives:
+- Understand the files needed for running LotuS2
+- Learn to use datasets from Zenodo in a Galaxy tool
+- Learn to upload data files to Galaxy
+- Learn to run the Galaxy LotuS2 tool and what the parameters mean
+- Understand the output files from LotuS2
+- Understand the structure of the mapping file needed by LotuS2 to link sample metadata to a pair of fastq files
+
+time_estimation: 3H
+
+key_points:
+- LotuS is a metagenomics tool for identifying species and {OTU}s and {ASV}s from sequencing data
+- Galaxy is an easy way to run LotuS2 in the cloud for bioinformatics beginners
+
+contributions:
+  authorship:
+    - sujaikumar
+  editing:
+    - bethanmanley
+
+---
+
+# Introduction
+
+This tutorial is for you if you are a mycologist using metabarcoding data to understand the fungal composition of soil samples. In particular, this tutorial and its downstream applications will be of most interest to mycorrhizal scientists. {SPUN} uses Illumina sequencing of environmental {eDNA} from soil samples to identify mycorrhizal fungi. At {SPUN}, we use both {ITS2} and {SSU} amplicon sequencing to identify ectomycorrhizal and arbuscular mycorrhizal fungi, respectively. This tutorial focuses on the {ITS2} sequencing used to identify {MF}.
+
+We do this by using a specific primer pair that amplifies just the {ITS2} region of the ribosomal DNA present in a soil sample. This tutorial covers data for an {ITS2} amplicon produced using the ITS3F and ITS4R primer set and sequenced using Illumina sequencing, which generates paired-end data. In this case, the example data has been generated using a NextSeq 2000, using 2x300bp chemistry.
+
+This tutorial concentrates on the bioinformatics part of the process, i.e. the steps we need to do **after** we get data back from the sequencer. Starting from some example fastq files, we demonstrate how to upload data, run LotuS2, and examine the output files. We end with a brief description of what we can do with these output files.
+
+We use LotuS2 at {SPUN} because we have found that it performs best out of all the tools we tried for identifying {MF} from sequencing data {% cite ozkurt2022 %}.
```

Comment:
you could also mention their own benchmark which outperformed most up-to-date amplicon tools using default parameters